controller/controller_server/utils.py

```diff
@@ -171,21 +206,20 @@ def validate_expand_volume_request(request):
     else:
         raise ValidationException(messages.no_capacity_range_message)
 
-    validate_secret(request.secrets)
+    validate_secrets(request.secrets)
+
+    if request.volume_capability:
+        logger.debug("validating volume capabilities")
```

Comment:
Remove this line since we log this message in the function below

Reply:
done